README.md

```diff
@@ -1 +1,96 @@
 # django-model-logging
+
+Logging for changes made to Django model instances.
+
+## Installation
+
+`pip install django-model-logging`
+
+Add `model_logging` to your `INSTALLED_APPS`.
+
+This library uses [`django-pgcrypto-fields`](https://github.com/incuna/django-pgcrypto-fields),
+which means you will need PGP public and private keys.  See the `settings.configure()`
+block in `model_logging/tests/run.py` for an (extremely insecure) example.
+
+## Usage
+
+#### Low-level use
+
+```
```

Comment:
can we enable syntax highlighting by adding `python` after the backticks please?